adapters/zio-http/src/main/scala/caliban/ZHttpAdapter.scala

```diff
@@ -50,8 +57,9 @@ object ZHttpAdapter {
   object Callbacks {
     def empty[R, E] = Callbacks[R, E](None, None)
 
-    def init[R, E](f: io.circe.Json => ZIO[R, E, Any]): Callbacks[R, E]               = Callbacks(Some(f), None)
-    def message[R, E](f: ZStream[R, E, Text] => ZStream[R, E, Text]): Callbacks[R, E] = Callbacks(None, Some(f))
+    def init[R, E](f: io.circe.Json => ZIO[R, E, Any]): Callbacks[R, E]               = Callbacks(Some(f), None, None)
```

Comment:
Can you add some scaladoc on these? It's not trivial so explanations would help.

Reply:
Sure! I added an attempt at explanations, let me know what you think :)